openapi.yaml

```diff
@@ -572,12 +573,17 @@ definitions:
                 $ref: '#/definitions/MinimumBet'
               maximumBet:
                 $ref: '#/definitions/MaximumBet'
+              round:
+                $ref: '#/definitions/Round'
+              betPool:
+                $ref: '#/definitions/BetPool'
```

Comment:
If `betPool` is not a required property, you should specify a default value https://swagger.io/docs/specification/2-0/describing-parameters/